src/main/java/cubicchunks/worldgen/generator/custom/CustomGeneratorSettings.java

```diff
@@ -85,6 +88,13 @@ public CustomGeneratorSettings() {
 
 	}
 
+	public BiomeBlockReplacerConfig createBiomeBlockReplacerConfig() {
+		BiomeBlockReplacerConfig conf = new BiomeBlockReplacerConfig();
+		conf.fillDefaults();
+		conf.set(CubicChunks.MODID, "ocean_height", this.waterLevel);
```

Comment:
Since ocean can be configured to be any block, not just water, I will change it to ocean level instead of water level.